src/mapml-viewer.js

```diff
@@ -986,7 +989,256 @@ export class HTMLMapmlViewerElement extends HTMLElement {
       }
     });
   }
+  matchMedia(query) {
+    // useful features for maps: prefers-color-scheme, prefers-lang, projection, zoom, extent
+    const parsedQuery = parseMediaQueryList(query);
+
+    // less obviously useful: aspect-ratio, orientation, (device) resolution, overflow-block, overflow-inline
+
+    const map = this;
+    const features = {
+      'prefers-lang': {
+        type: 'discrete',
+        get values() {
+          return [navigator.language.substring(0, 2)];
+        }
+      },
+      'map-projection': {
+        type: 'discrete',
+        get values() {
+          return [map.projection.toLowerCase()];
+        }
+      },
+      'map-zoom': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: false,
+        canBeZero: true,
+        get extraValues() {
```

Comment:
Why is this `extraValues`, while all the others are just `values`? Did not find this being used anywhere

Reply:
Good catch, I think it can be removed... tbd